### PR TITLE
DO NOT MERGE: Test of automated update Spring Boot to 1.5.14

### PR DESCRIPTION
**Pull request by metacosm**

### CODE REVIEW

**Comment by metacosm** [comment]
Should be good now. Sorry for the PR spam… :(

File: spring-boot/current-redhat/cache/booster.yaml [hunk 1]
```diff
@@ -2,13 +2,13 @@ metadata:
...
     launcher:
       runsOn:
-        - '!starter'
+      - '!starter'
 name: Spring Boot - Cache Example
 description: Booster to demonstrate how to use a cache server Spring Boot on OpenShift.
 source:
   git:
     url: https://github.com/snowdrop/spring-boot-cache-booster
-    ref: redhat
```
Comment (gytis) [inline spring-boot/current-redhat/cache/booster.yaml]: Is it correct to update untagged boosters?

File: metadata.yaml [hunk 2]
```diff
@@ -109,22 +248,45 @@ runtimes:
...
   versions:
   - id: current-community
-    name: 1.5.13.RELEASE (Community)
+    name: 1.5.14.RELEASE ( Community )
   - id: current-redhat
-    name: 1.5.13.RELEASE (RHOAR)
+    name: 1.5.14.RELEASE ( RHOAR )
   - id: current-osio
-    name: 1.5.12.RELEASE (OSIO)
+    name: 1.5.14.RELEASE ( OSIO )
```
Comment (gytis) [inline metadata.yaml]: These spaces look weird but I guess this is done automatically and is not configurable?

File: metadata.yaml [hunk 3]
```diff
@@ -83,19 +109,132 @@ runtimes:
     name: 7.0.0 (Red Hat Fuse)
 - id: nodejs
   name: Node.js
-  description: A JavaScript runtime built on Chrome's V8 JavaScript engine, using an event-driven, non-blocking I/O model for lightweight efficiency.
```
Comment (gytis) [inline metadata.yaml]: Is it ok to modify sections of other runtimes?

**Comment by gytis** [comment]
@metacosm I don't know how the automation was implemented, so just checking. In this PR all boosters were updated to the same version. However, in reality that is not the case. Will the script be able to handle different versions or branches where there is not tag?

File: metadata.yaml (hunk 3 above)
Comment (metacosm) [reply metadata.yaml]: Ideally, no, but the `yq` tool used to manipulate YAML does this :(

File: metadata.yaml (hunk 2 above)
Comment (metacosm) [reply metadata.yaml]: Actually, this is a mistake on my part that I have fixed after that PR :)

File: spring-boot/current-redhat/cache/booster.yaml (hunk 1 above)
Comment (metacosm) [reply spring-boot/current-redhat/cache/booster.yaml]: I'm actually not sure why that booster was done that way. It was the only one using the `environments` section. Switching from branch head to tag is better, in my opinion.

File: metadata.yaml (hunk 3 above)
Comment (geoand) [reply metadata.yaml]: I don't know how it;s implemented (need to look into it), but Atomist supposedly can perform these kinds of change without modifying the doc.

Our current tooling makes such changes inevitable

File: spring-boot/current-redhat/cache/booster.yaml (hunk 1 above)
Comment (geoand) [reply spring-boot/current-redhat/cache/booster.yaml]: I also don't understand this change

**Comment by metacosm** [comment]
@gytis Yes, this PR was created using a fake release number but the script actually takes the latest tag version, though it only updates the catalog if the SB version for the new tag is more recent than the one that's defined in the `booster.yaml`. I guess that could be improved to check for individual booster version as well. However, in this instance, since the update to `1.5.13` had already been merged, the script would have resulted in no significant change (apart from reformatting)… so I had to use a fake release version to make it do something so the output could be checked here.

**Comment by gytis** [comment]
So the script is not going to be used in a case where minor booster versions are updated (i.e. 1.5.13-x)?

File: spring-boot/current-redhat/cache/booster.yaml (hunk 1 above)
Comment (geoand) [reply spring-boot/current-redhat/cache/booster.yaml]: Does the change in `master` perhaps fix the issue?

Comment (geoand) [reply spring-boot/current-redhat/cache/booster.yaml]: I noticed that this PR conflicts with master on this file

**Comment by metacosm** [comment]
Not right now, but it should, I'll add an issue for it.

File: spring-boot/current-redhat/cache/booster.yaml (hunk 1 above)
Comment (metacosm) [reply spring-boot/current-redhat/cache/booster.yaml]: Ah yes, this PR depends on the change in master

**Comment by metacosm** [comment]
Let's close this PR and revisit for the next release :)

**Pull request closed without merging**